en/docs/chapter_sorting/merge_sort.md

```diff
@@ -46,10 +46,10 @@ The "merge phase" combines the left and right sub-arrays into a single ordered a
 === "<10>"
     ![merge_sort_step10](merge_sort.assets/merge_sort_step10.png)
 
-It is observed that the order of recursion in merge sort is consistent with the post-order traversal of a binary tree.
+It can be observed that the order of recursion in merge sort is consistent with the sub-array traversal of a binary tree.
 
-- **Post-order traversal**: First recursively traverse the left subtree, then the right subtree, and finally handle the root node.
-- **Merge sort**: First recursively handle the left sub-array, then the right sub-array, and finally perform the merge.
+- **sub-array traversal**: First recursively traverse the left subtree, then the right subtree, and finally process the root node.
```

Comment:
Did you replace "postorder" with "sub-array"?

Reply:
Sorry for the mistake, I have already fix this